### ISSUE TITLE
"span={6}" issue on Exchange.js Deep Dive   

### ISSUE THREAD

**Issue by vvronskyFX** [adrianhajdin/project_cryptoverse#38]
I was getting this issue with alignment of values: 
![image](https://user-images.githubusercontent.com/64465297/147809628-85b31d82-225e-48be-a308-03ec0bce45d1.png)

`span={6} `seemed to be the issue.

Here's what the code can be. Not perfect but it's functional!
![image](https://user-images.githubusercontent.com/64465297/147809676-ccb001e9-a1da-4d7b-92b3-f83f3b610175.png)

Change out `span={6}` to `style={{ width: "18.9vw" }}` inside the <Col />

See new layout w/ iPhone and Desktop:
![image](https://user-images.githubusercontent.com/64465297/147809709-86366bc0-3be5-410f-af62-fad5bd231182.png)

![image](https://user-images.githubusercontent.com/64465297/147809714-b8f7002a-8505-4362-8549-d858b5f1519f.png)

**Comment by vvronskyFX** [comment]
It's still a bit buggy on a responsive webpage. Please share & add in with any more efficient functionality. :)

**Comment by web-galtsov** [comment]
<Row >
                <Col span={6}>Exchanges</Col>
                <Col span={6} style={{textAlign:"center"}}>24h Trade Volume</Col>
                <Col span={6} style={{textAlign:"center"}}>Markets</Col>
                <Col span={6} style={{textAlign:"center"}}>Change</Col>
            </Row>
            <Row>
                 {exchangesList.map((exchange) => (
          <Col span={24}>
            <Collapse>
              <Panel
                key={exchange.uuid}
                showArrow={false}
                header={(
                  <Row key={exchange.uuid} style={{ width: "100%" }}>
                    <Col span={6}>
                      <Text><strong>{exchange.rank}.</strong></Text>
                      <Avatar className="exchange-image" src={exchange.iconUrl} />
                      <Text><strong>{exchange.name}</strong></Text>
                    </Col>
                  <Col span={6}>${millify(exchange.volume)}</Col>
                    <Col span={6} style={{textAlign:"center"}}>{millify(exchange.numberOfMarkets)}</Col>
                    <Col span={6} style={{textAlign:"center"}}>{millify(exchange.marketShare)}%</Col>
                  </Row>
                  )}
              >
                {HTMLReactParser(exchange.description || '')}
              </Panel>
            </Collapse>
          </Col>
        ))}
            </Row>

**Comment by web-galtsov** [comment]
<Row >
                <Col span={6}>Exchanges</Col>
                <Col span={6} style={{textAlign:"center"}}>24h Trade Volume</Col>
                <Col span={6} style={{textAlign:"center"}}>Markets</Col>
                <Col span={6} style={{textAlign:"center"}}>Change</Col>
            </Row>
            <Row>
                 {exchangesList.map((exchange) => (
          <Col span={24} key={exchange.uuid}>
            <Collapse>
              <Panel
                key={exchange.uuid}
                showArrow={false}
                header={(
                  <Row key={exchange.uuid} style={{ width: "100%" }}>
                    <Col span={6} >
                      <Text><strong>{exchange.rank}.</strong></Text>
                      <Avatar className="exchange-image" src={exchange.iconUrl} />
                      <Text><strong>{exchange.name}</strong></Text>
                    </Col>

                    <Col span={6} style={{textAlign:"center"}}>${millify(exchange.volume)}</Col>
                    <Col span={6} style={{textAlign:"center"}}>{millify(exchange.numberOfMarkets)}</Col>
                    <Col span={6} style={{textAlign:"center"}}>{millify(exchange.marketShare)}%</Col>
                  </Row>
                  )}
              >
                {HTMLReactParser(exchange.description || '')}
              </Panel>
            </Collapse>
          </Col>
        ))}
            </Row>

**Comment by vedanksingh** [comment]
To acces exchange do we have to take permium plan

**Comment by vedanksingh** [comment]
>    <Row >
>                 <Col span={6}>Exchanges</Col>
>                 <Col span={6} style={{textAlign:"center"}}>24h Trade Volume</Col>
>                 <Col span={6} style={{textAlign:"center"}}>Markets</Col>
>                 <Col span={6} style={{textAlign:"center"}}>Change</Col>
>             </Row>
>             <Row>
>                  {exchangesList.map((exchange) => (
>           <Col span={24} key={exchange.uuid}>
>             <Collapse>
>               <Panel
>                 key={exchange.uuid}
>                 showArrow={false}
>                 header={(
>                   <Row key={exchange.uuid} style={{ width: "100%" }}>
>                     <Col span={6} >
>                       <Text><strong>{exchange.rank}.</strong></Text>
>                       <Avatar className="exchange-image" src={exchange.iconUrl} />
>                       <Text><strong>{exchange.name}</strong></Text>
>                     </Col>
> 
>                     <Col span={6} style={{textAlign:"center"}}>${millify(exchange.volume)}</Col>
>                     <Col span={6} style={{textAlign:"center"}}>{millify(exchange.numberOfMarkets)}</Col>
>                     <Col span={6} style={{textAlign:"center"}}>{millify(exchange.marketShare)}%</Col>
>                   </Row>
>                   )}
>               >
>                 {HTMLReactParser(exchange.description || '')}
>               </Panel>
>             </Collapse>
>           </Col>
>         ))}
>             </Row>

Have you purchase tha premium plan ?

**Comment by web-galtsov** [comment]
Yes I did, but when I look at the .JSON I don't see the description field
